server/nginx.conf

```diff
@@ -0,0 +1,23 @@
+server {
+	location /upload {
+		client_body_temp_path      /tmp;
+		client_body_in_file_only   on;
+		client_body_buffer_size    1K;
+		client_max_body_size       1000M;
```

Comment:
We should shrink this to add some minimal protection against getting DOS'd. 50 MB seems like a reasonable limit to me.